BluetoothLEExplorer/BluetoothLEExplorer/Views/VirtualKeyboardPage.xaml

```diff
@@ -0,0 +1,115 @@
+                    <VisualState.Setters>
+                        <!--  TODO: change properties for wide view  -->
+                    </VisualState.Setters>
+                </VisualState>
+            </VisualStateGroup>
+        </VisualStateManager.VisualStateGroups>
+
+        <controls:PageHeader x:Name="pageHeader" RelativePanel.AlignLeftWithPanel="True"
+                             RelativePanel.AlignRightWithPanel="True"
+                             RelativePanel.AlignTopWithPanel="True" Text="Virtual Keyboard">
+
+            <!--  secondary commands  -->
+            <controls:PageHeader.SecondaryCommands>
+                <AppBarButton Click="{x:Bind ViewModel.GotoSettings}" Label="Settings" />
+                <AppBarButton Click="{x:Bind ViewModel.GotoPrivacy}" Label="Privacy" />
+                <AppBarButton Click="{x:Bind ViewModel.GotoAbout}" Label="About" />
+            </controls:PageHeader.SecondaryCommands>
+
+        </controls:PageHeader>
+
+        <RelativePanel RelativePanel.Below="pageHeader" Padding="10" Visibility="{x:Bind ViewModel.IsPeripheralRoleSupported, Converter={StaticResource VisibleWhenFalseConverter}}">
+            <TextBlock Text="Peripheral mode is not supported on this device" />
+        </RelativePanel>
+
+        <RelativePanel RelativePanel.Below="pageHeader" Padding="10" Visibility="{x:Bind ViewModel.IsPeripheralRoleSupported, Converter={StaticResource VisibleWhenTrueConverter}}">
+            <StackPanel Orientation="Vertical">
+                <TextBlock Text="{x:Bind ViewModel.ErrorText, Mode=OneWay}" />
+                <StackPanel Orientation="Horizontal">
+                    <ProgressRing VerticalAlignment="Top" IsActive="True" Visibility="{x:Bind ViewModel.IsSettingUp, Mode=OneWay, Converter={StaticResource VisibleWhenTrueConverter}}" HorizontalAlignment="Left" />
+                    <TextBlock Text=" " Width="20" />
+                    <ToggleSwitch x:Name="VirtualKeyboardSwitch"
+                                  Header="Discoverable and Connectable"
+                                  IsOn="{x:Bind ViewModel.IsKeyboardEnabled, Mode=TwoWay}"
+                                  IsEnabled="{x:Bind ViewModel.IsSettingUp.Equals(x:False), Mode=OneWay}"
+                                  IsFocusEngagementEnabled="False"
+                                  AllowFocusOnInteraction="False"
+                                  AllowFocusWhenDisabled="False"
+                                  IsTabStop="False" />
+                </StackPanel>
+                <TextBlock Visibility="{Binding IsOn, ElementName=VirtualKeyboardSwitch, Converter={StaticResource VisibleWhenTrueConverter}}"
+                           Text="While this application is in focus, keystrokes from your physical keyboard will be routed to any and all paired clients that are connected and subscribed to the HID notifications."
+                           TextWrapping="WrapWholeWords" />
+            </StackPanel>

```

Comment:
Add a note explaining the Android bug, specifically when they need to toggle the radio. If we have a public bug link they can upvote consider placing it here. :)

Reply:
Let me know if you think the wording now is sufficient.